### PR TITLE
[PRP-181] Set up testing for staff portal (redo)

### PR DESCRIPTION
**Pull request by rocketnova**
## Ticket

https://wicmtdp.atlassian.net/browse/PRP-181

- Configure Playwright for staff portal
- Create initial e2e test with screenshots
- Configure eslint
- Add npm scripts to update prisma, run lowdefy, run e2e

This PR configures the staff portal to be able to run Playwright for e2e tests. When you run `npm run e2e`, it will run the same pattern of commands as in #30. If Prisma files need to be updated to match current changes in the participant remix app, you can run `npm run prisma:prepare`.

Run `npm run e2e`

### CODE REVIEW

**Comment by rocketnova** [comment]
@microwavenby This PR replaces #29 which I broke so completely that there was no recovering it. Sorry to have you re-review, but it should be largely the same.

**Comment by rocketnova** [comment]
Oh I just realized I need to rework this for the e2e changes I made for the participant app.

File: staff/package.json [hunk 1]
```diff
@@ -2,14 +2,37 @@
   "name": "staff_portal",
   "private": true,
   "scripts": {
+    "dev": "npx lowdefy@3.23.3 dev",
+    "e2e": "docker compose -f docker-compose.e2e.yml up --build --wait database-e2e && docker compose -f docker-compose.e2e.yml up --build --wait lowdefy-e2e && docker compose -f docker-compose.e2e.yml run --rm playwright npx playwright test --retries=3 --reporter=list",
+    "e2e:update": "docker compose -f docker-compose.e2e.yml up --build --wait database-e2e && docker compose -f docker-compose.e2e.yml up --build --wait lowdefy-e2e && docker compose -f docker-compose.e2e.yml run --rm playwright npx playwright test --update-snapshots --reporter=list",
```
Comment (microwavenby) [inline staff/package.json:7]: maybe just a bash script? These are pretty wildly long commands!

File: staff/playwright.config.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+       * GitHub Actions runs on Linux, and we're faking those screenshots
+       * by copying the Darwin (macos) ones
+       */
+
+      /* Another option is to have a command to update the screenshots from Github with
+       * a specific command:
+       * https://github.com/basarat/demo-playwright-vrt/blob/main/.github/workflows/update-snapshots.yml
+       */
+      maxDiffPixelRatio: 0.01,
+    },
```
Comment (microwavenby) [inline staff/playwright.config.ts:34]: probably can skip this whole block rather than having a default drift now

**Review by microwavenby — APPROVED** [review]

File: staff/package.json (hunk 1 above)
Comment (rocketnova) [reply staff/package.json:7]: Follow on ticket: https://wicmtdp.atlassian.net/browse/PRP-256

File: staff/playwright.config.ts (hunk 2 above)
Comment (rocketnova) [reply staff/playwright.config.ts:34]: Follow on ticket: https://wicmtdp.atlassian.net/browse/PRP-256

**Pull request merged**